src/components/appheader.main.jsx

```diff
@@ -165,7 +165,18 @@ class AppHeaderMain extends React.Component {
                 <div className="search-icon" />
               </button>
             </div>
+            {(Config.b2b.enable && availability) && (cartData && cartData._additemstocartform) && (
```

Comment:
Moving these elements up appears to cause a regression in the mobile header layout:
![image](https://user-images.githubusercontent.com/39800563/56869436-b4434e00-69ce-11e9-8d81-3124fa1b3f5e.png)